packages/backend/src/config/app.test.ts

```diff
@@ -0,0 +1,94 @@
+import * as assert from 'assert'
+import * as crypto from 'crypto'
+import * as fs from 'fs'
+import { parseOrProvisionKey } from './app'
+
+describe('Config', (): void => {
+  describe('parseOrProvisionKey', (): void => {
+    const TMP_DIR = './tmp'
+    const PRIVATE_KEY_FILE = `${TMP_DIR}/private-key.pem`
+
+    beforeEach(async (): Promise<void> => {
+      fs.rmSync(TMP_DIR, { recursive: true, force: true })
+    })
+
+    afterEach(async (): Promise<void> => {
+      fs.rmSync(TMP_DIR, { recursive: true, force: true })
+    })
+
+    test('can provision key', async (): Promise<void> => {
+      expect(fs.existsSync(TMP_DIR)).toBe(false)
+      expect(fs.existsSync(PRIVATE_KEY_FILE)).toBe(false)
+      const { key, jwk } = parseOrProvisionKey(PRIVATE_KEY_FILE)
+      expect(key).toBeInstanceOf(crypto.KeyObject)
+      expect(jwk).toMatchObject({
```

Comment:
```suggestion
```suggestion
      expect(key).toMatchObject({
        asymmetricKeyType: 'ed25519',
        type: 'private'
      })
      expect(jwk).toEqual(key.export({ format: 'jwk' }))
      expect(jwk).toEqual({
```